app/auth/controllers.py

```diff
@@ -56,7 +56,8 @@ def callback_handling():
         user = User().create_or_update_user_from_callback(user_info.data)
         jwt_helper = JWT(app.config['API_KEY'], user.id)
         session.pop('github_token', None)
-        resp = make_response(render_template("dashboard.html", user=user,
+        g.current_user = user
+        resp = make_response(render_template("dashboard.html",
```

Comment:
You don't want to do the render here - you want to redirect to dashboard page (which is `/` for now)